src/app/core/components/header/header.component.ts

```diff
@@ -70,4 +70,8 @@ export class HeaderComponent implements OnInit {
   toggleNav() {
     this.isShown = !this.isShown;
   }
+
+  onSearchNavigation() {
```

Comment:
Why is this new function needed, rather than just calling toggleNav()?

Reply:
Excellent question. It isn't needed, so I removed it.